src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx

```diff
@@ -73,15 +75,21 @@ class ResponsiveTable extends Component {
     );
   };
 
+  onHeaderClick = (value, order) => () => {
+    // This replaces `this.props.onSort`.
+    this.props.onHeaderClick(value, order);
+  };
+
   render() {
     const { className, data, fields } = this.props;
     const headers = fields.map(this.renderHeader);
     const rows = data.map(this.renderRow);
+    const tableClass = classNames('va-sortable-table responsive', className);
 
     return (
-      <table className="responsive">
+      <table role="table" className={tableClass}>
         <thead>
-          <tr>{headers}</tr>
+          <tr role="row">{headers}</tr>
```

Comment:
You might want to get @jenstrickland to look at these... I don't think you need to include a `role` for table, tr and td elements.

Reply:
@Mottie We need to include the role for rows and cells because the table CSS display properties are being overwritten.

Reply:
@Mottie These were @jenstrickland 's suggestion, the table needs them on mobile because it uses `display:block`